src/components/icon-button/icon-button.pw.tsx

```diff
@@ -0,0 +1,232 @@
+import React from "react";
+import { test, expect } from "@playwright/experimental-ct-react17";
+import IconButtonComponent from "./component.test-pw";
+import { CHARACTERS } from "../../../playwright/support/constants";
+import { HooksConfig } from "../../../playwright";
+import { checkAccessibility } from "../../../playwright/support/helper";
+
+test.describe(
+  "check IconButton component focus outlines and border radius",
+  () => {
+    test(`should have the expected styling when the focusRedesignOptOut is false`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<IconButtonComponent aria-label={CHARACTERS.STANDARD} />);
```

Comment:
**question** (non-blocking): I can see we're passing an `aria-label` for these focus and radius tests, is there a specific reason for this, or can they be removed. I can see we do this in the cypress test too btw, so I understand why you've included them. Just could get rid if they're not neccessary